utils/templates/test_module.yml.in

```diff
@@ -0,0 +1,54 @@
+---
+- name: Test $name
+  hosts: ipaserver
+  become: true
+
+  tasks:
+
+  # CLEANUP TEST ITEMS
+
+  - name: Ensure $name NAME is absent
+    ipa$name:
+      name: NAME
```

Comment:
The `standard password` should be added here: `ipaadmin_password: SomeADMINpassword`.